### PR TITLE
Add plain javascript code for live-updating notification counts and lists

### LINKED ISSUE #71: JSON access to recent notifications
Some projects may find it useful to have an easy to drop in Javascript/JSON function with an appropriate updating field that fetches notifications on behalf of the user:

For example, inserting the code similar to the following into a template:

    {% load notifications_tags %}
    {% live_notify_icon %}

Would provide an auto-updating notification badge indicating there are unread notfications, without a page refresh.

Similarly, something like this:

    {% load notifications_tags %}
    {% live_notify_menu 5 %}

Would provide a simple `ul` menu with the 5 most recent unread notifications (with the number being configurable).

I've assigned this bug to me already and can do some work on it, and if there is interest we can look at pulling it in.

For reference, this would be done in a way that adds no extra requirements, and done using raw javascript so we don't force libraries like jquery onto users.

Any thoughts on this?

### PR DESCRIPTION
**Pull request by LegoStormtroopr**
The code in this pull requests adds code to easily insert javascript for handling live-updating notification lists and counts of *unread* messages only at this stage.

I'll leave it open for a few days for review and comment. At this stage the API and Javascript are designed to be minimal and not require additional javascript libraries.

Comments are appreciated at this stage. You can test this out by dropping `{% live_notify_badge %}` into a template, and then adding notifications, either through an app of via a django `./manage.py shell`.

This will close #71 once accepted.

### CODE REVIEW

**Comment by yangyubo** [comment]
Well done, @LegoStormtroopr 

But unfortunately, it breaks test cases on Python 3.4.x.

File: notifications/static/notifications/notify.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }
+}
+
+function fill_notification_list(data) {
+    var menu = document.getElementById(notify_menu_id);
+    if (menu) {
+        menu.innerHTML = "";
+        for (var i=0; i < data.unread_list.length; i++) {
+            var item = data.unread_list[i];
+            menu.innerHTML = menu.innerHTML + "<li>"+item.object+" "+item.verb+" "+item.subject+"</li>";
```
Comment (thetylerhayes) [inline notifications/static/notifications/notify.js]: I know this has already been merged, but where are `item.object` and `item.subject` in this repo's standard JSON response? Those aren't available in `data.unread_list` items AFAICT.

I'm testing out this repo right now and it's falling down here.

Screenshots may help:

screenshot 2015-11-10 16 57 21

screenshot 2015-11-10 16 58 17

**Comment by LegoStormtroopr** [comment]
@thetylerhayes as the person responsible for this fork, I've raised an issue on your behalf ( #82 ). I'll check this out during the week and see what I can sort out. Thanks for raising this.

**Comment by thetylerhayes** [comment]
:+1:

**Pull request merged**